src/shared/lib/GenericAssayUtils/MutationalSignaturesUtils.tsx

```diff
@@ -163,3 +166,35 @@ export function validateMutationalSignatureRawData(
     // we are expecting contribution and pvalue profiles are in pairs
     return _.every(profileIdsGroupByVersion, ids => ids.length === 2);
 }
+
+export function retrieveMutationalSignatureVersionFromData(
+    SignatureProfiles: any[]
+) {
+    let valueToSet: string = 'v2';
```

Comment:
use ```const``` unless you know for sure you are going to change the value later (unusual).